resources.go

```diff
@@ -99,6 +99,9 @@ func (a *App) createResources() error {
 			Args: fs.Args{"id": createArg(fs.TypeUint64, "The content ID")},
 		})).
 		Add(fs.NewResource("create", contentService.Create, &fs.Meta{Post: "/"})).
+		Add(fs.NewResource("bulk-update", contentService.BulkUpdate, &fs.Meta{
+			Put: "/bulk-update",
```

Comment:
We used `PUT /content/test/:id` for single record update, so I think we should use `PUT /content/test/update` for bulk update operator.